### PR TITLE
[zh]fix 2019 blogs: add slug and remove url

### PR DESCRIPTION
**Pull request by howieyuen**
related issue: https://github.com/kubernetes/website/issues/34487

### CODE REVIEW

File: content/zh-cn/blog/_posts/2019-04-26-latest-on-localization.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 layout: blog
 title: '如何参与 Kubernetes 文档的本地化工作'
 date: 2019-04-26
+slug:
```
Comment (mengjiao-liu) [inline content/zh-cn/blog/_posts/2019-04-26-latest-on-localization.md:5]: empty?

Comment (howieyuen) [reply content/zh-cn/blog/_posts/2019-04-26-latest-on-localization.md]: sorry for my carelessness

**Comment by howieyuen** [comment]
cc @tengqm

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**